gitbook/src/monitoring/monitoring.md

```diff
@@ -0,0 +1 @@
+# Monitoring
```

Comment:
we have `./cmd/prometheus/readme.md` - there are some default dashboards. But unfortunately we now spending very little time to support them. We using our internal Grafana board and very rare exporting it to Erigon's repo.